### PR TITLE
night mode fixes for candidate packages and reverse dependencies

### PR DESCRIPTION
**Pull request by peterbecich**
- candidate packages
Screenshot 2023-01-06 at 16-50-05 Checking and uploading candidates Hackage
Screenshot 2023-01-06 at 17-09-02 2captcha Haskell package for easy integration with the 2captcha API

### CODE REVIEW

**Comment by peterbecich** [comment]
Reverse dependencies; dark and light modes
Screenshot 2023-01-06 at 17-59-54 AesonBson
Screenshot 2023-01-06 at 18-00-16 AesonBson

File: datafiles/templates/packagePageAssets.st [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
   div#modal #content {
     border-radius: 8px;
-    background: #fff;
```
Comment (peterbecich) [inline datafiles/templates/packagePageAssets.st:23]: has to be removed, otherwise it takes precedence over `hackage.css`

**Comment by peterbecich** [comment]
@gbaz , please review, I will fix the quickjump issue separately: https://github.com/haskell/hackage-server/pull/1163

**Comment by gbaz** [comment]
I deployed these manually on prod and they look good so far!

**Comment by peterbecich** [comment]
Thanks!

**Comment by peterbecich** [comment]
Do you see this transparency here? I need to fix something
 https://hackage.haskell.org/package/hint
Screenshot 2023-01-06 at 20-55-59 hint

**Comment by peterbecich** [comment]
Still trying to repro locally, doesn't appear immediately
Screenshot 2023-01-06 at 21-04-46 BiobaseEnsembl

**Comment by gbaz** [comment]
I suspect just stale css. I purged the cache, but you may need a forced-reload.

**Comment by peterbecich** [comment]
you're correct, looks good now
Screenshot 2023-01-06 at 21-11-14 hint

**Comment by peterbecich** [comment]
combined with https://github.com/haskell/hackage-server/pull/1163/commits

**Pull request closed without merging**